main.py

```diff
@@ -614,6 +613,72 @@ async def filter_outlet(pipeline_id: str, form_data: FilterForm):
             detail=f"{str(e)}",
         )
 
+@app.post("/v1/{pipeline_id}/embedding")
```

Comment:
This should match the OpenAI API spec, so that providing `{pipeline_id}` is not required from the open webui.

Reply:
Do you want the vector store and file api instead?
[](https://platform.openai.com/docs/api-reference/vector-stores/create)
[](https://platform.openai.com/docs/api-reference/files/create)
We'd have to add a database to the project then.